db/mainphotos copy.sql

```diff
@@ -0,0 +1,74 @@
+CREATE SCHEMA IF NOT EXISTS mainphotos;
+
+CREATE TABLE mainphotos.neighborhoods (
+  neighborhood_id INT GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
+  neighborhood_name VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE mainphotos.property_listings (
+  listing_id INT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY NOT NULL,
+  neighborhood INT NOT NULL REFERENCES mainphotos.neighborhoods( neighborhood_id ) ON DELETE RESTRICT,
+  address VARCHAR(100) NOT NULL,
+  pricing DECIMAL(100,2) NOT NULL,
+  bedrooms DECIMAL(3,1) NOT NULL,
+  baths DECIMAL(3,1) NOT NULL,
+  sq_footage INT NOT NULL
+);
+
+CREATE TABLE mainphotos.image_urls (
+  image_id INT PRIMARY KEY NOT NULL GENERATED BY DEFAULT AS IDENTITY,
+  url VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE mainphotos.property_images (
+  id INT GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
+  listing_id INT NOT NULL REFERENCES mainphotos.property_listings(listing_id) ON DELETE CASCADE,
```

Comment:
Keep your NOT NULL declarations consistent. Seems like most of it is at the end so:
- `listing_id INT REFERENCES mainphotos.property_listings(listing_id) NOT NULL ON DELETE CASCADE`